.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,6 +14,7 @@ See also #23
 - [ ] `npm test` passes on your machine
 - [ ] New tests added or existing tests modified to cover all changes
 - [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
-- [ ] Related API Documentation was updated
+- [ ] API Documentation in code was updated
+- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
```

Comment:
The link is crafted in such way that it works inside pull request descriptions. It does not work when you preview the markdown file on its own.